### PR TITLE
General mapping bugfixes

### LINKED ISSUE #13321: The Human Harvest book in botany is blank.
(WEB REPORT BY: probe1 REMOTE: 172.93.109.202:7777)
That's The Human Harvest - From seed to market. It is a normal-sized item.

This book is completely blank!

### LINKED ISSUE #17979: Deff medbay juke has no nanovinyls in it.
#### Description of issue
the medbay juke on deff has no music for you to listen to like every other jukebox on other maps.

### LINKED ISSUE #19029: deff bar has no newscaster
(WEB REPORT BY: whoru REMOTE: 172.93.109.202:7777)
un fucking playable

### LINKED ISSUE #19481: meta medbay access
(WEB REPORT BY: tellossee REMOTE: 172.93.109.202:7777)
# Revision
aad64615f32b5e034a1f0cdad451c5349b7472fd

# Description
parameds can't access the doors leading to the mourge
# Steps to Reproduce
try to go through to the mourge through medbay
# What you Expected
to get into the fucking mourge
# What Actually Happened
access denied
-------
Also paramed should have access to the cloner to speed clone.

### LINKED ISSUE #19674: Packed Station TEG room vulnerable to radiation and gravity pull from singulo
#### Description of issue

TEG room on Packed Station is vulnerable to singularity interference.
Items on the floor in the marked squares are pulled to the lower left corner, and people not wearing radiation suits receive high doses of radiation when standing in that area. 

image

#### Difference between expected and actual behavior

No part of the room should be getting pulled or irradiated.

#### Steps to reproduce 

Start singularity on Packed Station
In marked squares, stand or place items on floor. 
Receive rads, have items pulled from hands and see stuff get dragged around on the floor.

#### Length of time in which bug has been known to occur

No idea

#### Server revision

be82080b47abc2f43ff9bc1e00d598ee53c3dd6a 

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [X] Issue could be reproduced at least once
- [X] Issue could be reproduced by different players
- [ ] Issue could be reproduced in multiple rounds <-- did not test
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### LINKED ISSUE #20008: missing pipe under deff morgue door
the door connecting deff morgue to medbay lacks a distro pipe underneath

### LINKED ISSUE #20152: fire lock is wrong on deff morgue
#### Description of issue

firelock facing the wrong way

#### Difference between expected and actual behavior

wrong way

#### Steps to reproduce 

lookatit

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [ ] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [ ] Issue could be reproduced in multiple rounds
- [ ] Issue happened in a recent (less than 7 days ago) round
- [ ] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### LINKED ISSUE #20256: roidstation and bagelstation z-level2 cause the server to crash when trying to generate a map render
Not an issue with other maps

Thankfully it's the only zlevel not displayed in the web renders so it's not necessary but it means you can't use the "All zlevels" setting

### LINKED ISSUE #20421: Box Station Research Shuttle freaks the fuck out when airlock is opened
(WEB REPORT BY: mask2697 REMOTE: 172.93.109.202:7777)
# Revision
23d284d870c417754ce95bf1e927aad92ff47d91

# Description
When the airlock to space is opened, regardless of whether or not the shuttle is docked, the emergency airlocks immediately fall as if it has vented to space, with a corresponding alert being issued.  this generally occurs when the shuttle is docked and the air is safe.
# Steps to Reproduce

Dock research shuttle on station, open the airlock
# What you Expected

to embark/disembark

# What Actually Happened

Airlocks down PANIC MODE

### PR DESCRIPTION
**Pull request by CptWad**
I am procrastinating doing other things, obviously
deff:
fixes #19029
also adds a few newscasters to other areas lacking them
fixes #20152
fixes #20008
Box:
fixes #20421
meta: 
fixes #19481
packed:
fixes #19674
fixes #13321

possibly fixes #20256
I removed some null turfs so thats probably what went wrong

:cl:
 * bugfix: fixed bugs across several maps. If you find a bug please report it. If its not on git in ain't gettin' fixed.
 * rscadd: adds a throne to the bridge on bagel. Just because I was feeling fly.
 * rscadd: The empty botany book is now filled with words. Educate yourself on Dionae today!

### CODE REVIEW

**Comment by Shadowmech88** [comment]
@CptWad since you're touching all the maps, could you also fix #13321 by removing said book from all the maps?

**Comment by CptWad** [comment]
why not just fill it with words?

**Comment by CptWad** [comment]
I guess we can replace it on how dionae work

**Comment by SonixApache** [comment]
did you (or someone else) happen to fix the rogue disposal pipe in the equipment room in deff's brig?

**Comment by Shadowmech88** [comment]
I'm not sure dionae are worthy of a book. It's pretty much just "plant seed, harvest nymph" isn't it?

**Comment by CptWad** [comment]
@SonixApache done

I mean its just code right?
here ill fill it in a second

**Comment by CptWad** [comment]
uh oh was that a failed compile

**Comment by CptWad** [comment]
Whats the map diff renders? Whats wrong?

**Comment by Exxion** [comment]
I think it was an SQL error or something
I'm not really sure what MapDiffBot uses SQL for though

**Comment by CptWad** [comment]
>Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---> MySql.Data.MySqlClient.MySqlException: Error submitting 30MB packet; ensure 'max_allowed_packet' is greater than 30MB. ---> System.Net.Sockets.SocketException: Broken pipe
   at MySqlConnector.Utilities.SocketAwaitable.GetResult() in C:\projects\mysqlconnector\src\MySqlConnector\Utilities\SocketAwaitable.cs:line 30

**Comment by Exxion** [comment]
I assume it's the images, but I really don't know for sure
Either way it's probably not a problem with your changes

**Comment by CptWad** [comment]
So because I am changing roughly 100 lines of code across several blocks of code its angry?
whatever

**Comment by Exxion** [comment]
It has to actually render the maps into images and send the images
I think
I've never actually looked at its output

File: maps/bagelstation.dmm [hunk 1]
```diff
@@ -12937,6 +12937,7 @@
 "eOO" = (/obj/structure/cable/orange{d1 = 4; d2 = 8; icon_state = "4-8"},/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 9},/turf/simulated/floor{icon_state = "dark"},/area/prison/cell_block/A)
 "eOP" = (/obj/structure/bed/chair/shuttle{dir = 8},/obj/effect/decal/cleanable/dirt,/turf/simulated/floor/engine,/area/shuttle/salvage/start)
 "eOQ" = (/obj/structure/bed/chair/shuttle{dir = 1},/obj/effect/decal/cleanable/dirt,/turf/simulated/floor/engine,/area/shuttle/salvage/start)
+"eOR" = (/turf/space,/turf/simulated/shuttle/wall{icon_state = "swall_s9"},/area/centcom/evac)
```
Comment (Exxion) [inline maps/bagelstation.dmm:12940]: Here

File: maps/bagelstation.dmm [hunk 2]
```diff
@@ -8745,7 +8745,7 @@
 "dmi" = (/obj/structure/bed/chair/shuttle/red{dir = 1},/turf/simulated/shuttle/floor{icon_state = "floor4"},/area/shuttle/specops/centcom)
 "dmj" = (/obj/machinery/light,/obj/structure/bed/chair/shuttle/red{dir = 1},/turf/simulated/shuttle/floor{icon_state = "floor4"},/area/shuttle/specops/centcom)
 "dmk" = (/turf/simulated/shuttle/wall{icon_state = "swall7"},/area/centcom/evac)
-"dml" = (/turf/simulated/shuttle/wall{icon_state = "swall_s9"},/area/centcom/evac)
+"dml" = (/turf/space,/turf/simulated/shuttle/wall{icon_state = "swall_s5"},/area/centcom/evac)
```
Comment (Exxion) [inline maps/bagelstation.dmm:8748]: Here

File: maps/bagelstation.dmm [hunk 3]
```diff
@@ -8734,7 +8734,7 @@
 "dlX" = (/turf/simulated/floor{icon_state = "green"; dir = 4},/area/tdome)
 "dlY" = (/obj/effect/landmark{name = "tdome1"},/turf/unsimulated/floor{name = "plating"},/area/tdome/tdome1)
 "dlZ" = (/obj/structure/rack,/obj/item/clothing/under/color/green,/obj/item/clothing/shoes/brown,/obj/item/clothing/suit/armor/tdome/green,/obj/item/clothing/head/helmet/thunderdome,/obj/item/weapon/melee/baton/loaded,/obj/item/weapon/melee/energy/sword/green,/turf/unsimulated/floor{icon_state = "dark"},/area/tdome)
-"dma" = (/turf/simulated/shuttle/wall{icon_state = "swall_s5"},/area/centcom/evac)
+"dma" = (/turf/space,/turf/simulated/shuttle/wall{icon_state = "swall_s10"},/area/centcom/evac)
```
Comment (Exxion) [inline maps/bagelstation.dmm:8737]: Here

File: maps/bagelstation.dmm [hunk 4]
```diff
@@ -8692,14 +8692,14 @@
...
-"dlk" = (/turf/simulated/shuttle/wall{icon_state = "swall_s6"},/area/centcom/evac)
+"dlk" = (/obj/structure/bed/chair/wood/throne{dir = 4},/turf/simulated/floor{icon_state = "blue"; dir = 4},/area/bridge)
 "dll" = (/obj/structure/shuttle/engine/propulsion/right{dir = 1},/turf/space,/area/centcom/evac)
 "dlm" = (/obj/structure/shuttle/engine/propulsion{dir = 1},/turf/space,/area/centcom/evac)
 "dln" = (/obj/structure/shuttle/engine/propulsion/left{dir = 1},/turf/space,/area/centcom/evac)
 "dlo" = (/turf/simulated/shuttle/wall{icon_state = "swall14"},/area/centcom/evac)
 "dlp" = (/obj/machinery/light{dir = 1},/obj/structure/bed/chair/shuttle/red{dir = 4},/turf/simulated/shuttle/floor{icon_state = "floor4"},/area/shuttle/specops/centcom)
 "dlq" = (/turf/simulated/shuttle/wall{icon_state = "swall12"},/area/centcom/evac)
-"dlr" = (/turf/simulated/shuttle/wall{icon_state = "swall_s10"},/area/centcom/evac)
+"dlr" = (/turf/space,/turf/simulated/shuttle/wall{icon_state = "swall_s6"},/area/centcom/evac)
```
Comment (Exxion) [inline maps/bagelstation.dmm:8702]: Here

File: maps/RoidStation.dmm [hunk 5]
```diff
@@ -7591,6 +7591,10 @@
 "cPY" = (/obj/machinery/atmospherics/pipe/manifold/cyan/visible{dir = 8},/turf/simulated/floor,/area/engineering/atmos)
 "cPZ" = (/obj/structure/lattice,/obj/machinery/light/small{dir = 8; flickering = 1},/turf/space,/area)
 "cQa" = (/obj/structure/lattice,/obj/machinery/light/small{dir = 8; flickering = 0},/turf/space,/area)
+"cQb" = (/turf/space,/turf/simulated/shuttle/wall{icon_state = "swall_s6"},/area/centcom/evac)
+"cQc" = (/turf/space,/turf/simulated/shuttle/wall{icon_state = "swall_s10"},/area/centcom/evac)
+"cQd" = (/turf/space,/turf/simulated/shuttle/wall{icon_state = "swall_s5"},/area/centcom/evac)
+"cQe" = (/turf/space,/turf/simulated/shuttle/wall{icon_state = "swall_s9"},/area/centcom/evac)
```
Comment (Exxion) [inline maps/RoidStation.dmm:7597]: Here and the three previous lines

**Comment by Shadowmech88** [comment]
I didn't even know double turfs was possible.

**Comment by Exxion** [comment]
It isn't. You can place multiple turfs on the same spot in the mapper, but only one of them will actually be there in-game. The rest are added as underlays to the one that actually appears, if memory serves.

**Comment by CptWad** [comment]
I just do what my predesessors before me have done
else it breaks everything
Is there anything I should do or...?

**Comment by DamianX** [comment]
Is there any reason for the shuttle turf to not automatically add a space underlay to itself? By the way, this can definitely be dealt with in a different PR.

**Comment by SarahJohnson489** [comment]
@CptWad Got time for some low air turfs on box tox launch causing alarms?

tox air loss

**Comment by SarahJohnson489** [comment]
Double firelock on the sci main to research shuttle airlock. Drops a firelock parts on the ground on server startup.
untitled

**Comment by SarahJohnson489** [comment]
Missing disposal pipe under the glass to pen 6
untitled

**Comment by CptWad** [comment]
>Is there any reason for the shuttle turf to not automatically add a space underlay to itself? By the way, this can definitely be dealt with in a different PR.

It could but its not causing the failed render error in fact 
it should fix #20256 due to it being the ONLY difference between other maps
Usually, mas will fit space under the invisible stuff, HOWEVER the reading tools we have probably dont read that properly and throw errors. 
The ORIGINAL pr was fine and had no unsuccessful checks. I am guessing its throwing a hissy fit over something irrelevant since I literally cant find anything wrong

@SarahJohnson489 next commit

File: code/game/objects/items/weapons/manuals.dm [hunk 6]
```diff
@@ -127,24 +127,22 @@
...
+				What are Dionae? These fun little plant people are something that every botanist needs to understand.
 				<p>
-				<ol>
-				<li>Find a dead person who is in need of cloning. </li>
-				<li>Take a blood sample with a syringe. </li>
-				<li>Inject a seed pack with the blood sample. </li>
-				<li>Plant the seeds. </li>
-				<li>Tend to the plants water and nutrition levels until it is time to harvest the cloned human.</li>
-				</ol>
+				Dionae nypmh pods will produce a single nyph. Most of the time the nymph dies out producing more seeds for replanting but when a nymph fully realizes it will start crawling about and now has some new abilties!<br>
```
Comment (DamianX) [inline code/game/objects/items/weapons/manuals.dm]: nyph

File: code/game/objects/items/weapons/manuals.dm [hunk 7]
```diff
@@ -127,24 +127,22 @@
...
+				Dionae can fertilze your crop, eat weeds and help around if they are willing.<br>
+				Amoung the Dionae nymphs needs are blood collection. They are likely to flick out their feelers at new people around itself nicking a sample of their blood. <br>
```
Comment (DamianX) [inline code/game/objects/items/weapons/manuals.dm]: amoung

Comment (DamianX) [inline code/game/objects/items/weapons/manuals.dm]: `among ... are` vs `among ... is`?

File: code/game/objects/items/weapons/manuals.dm [hunk 8]
```diff
@@ -127,24 +127,22 @@
...
+				This blood sample is used by dionae biology to gather the biological symbols from its source to garnish itself with new memories. They tend to mimic anything from their vocal patterns, languages, attitudes and opinons.<br>
```
Comment (DamianX) [inline code/game/objects/items/weapons/manuals.dm]: You used capital D in Dionae everywhere but here

File: code/game/objects/items/weapons/manuals.dm [hunk 9]
```diff
@@ -127,24 +127,22 @@
...
+				However this does not mean that they nessisarily need to follow your every whim. They tend to form their own wants and needs, and your green children do grow up fast. With enough collected memory they will evolve into adult Dionae.<br>
```
Comment (DamianX) [inline code/game/objects/items/weapons/manuals.dm]: nessisarily

File: code/game/objects/items/weapons/manuals.dm (hunk 6 above)
Comment (CptWad) [reply code/game/objects/items/weapons/manuals.dm]: https://en.wikipedia.org/wiki/Nymph

File: code/game/objects/items/weapons/manuals.dm [hunk 10]
```diff
@@ -127,24 +127,22 @@
...
-				<li>Take a blood sample with a syringe. </li>
-				<li>Inject a seed pack with the blood sample. </li>
-				<li>Plant the seeds. </li>
-				<li>Tend to the plants water and nutrition levels until it is time to harvest the cloned human.</li>
-				</ol>
+				Dionae nypmh pods will produce a single nymph. Most of the time the nymph dies out producing more seeds for replanting but when a nymph fully realizes it will start crawling about and now has some new abilties!<br>
+				Dionae can fertilze your crop, eat weeds and help around if they are willing.<br>
+				Among the Dionae nymphs needs is blood collection. They are likely to flick out their feelers at new people around itself nicking a sample of their blood. <br>
+				This blood sample is used by Dionae biology to gather the biological symbols from its source to garnish itself with new memories. They tend to mimic anything from their vocal patterns, languages, attitudes and opinons.<br>
+				However this does not mean that they necissarily need to follow your every whim. They tend to form their own wants and needs, and your green children do grow up fast. With enough collected memory they will evolve into adult Dionae.<br>
```
Comment (DamianX) [inline code/game/objects/items/weapons/manuals.dm]: it's fucking `necessarily`

**Pull request merged**